crates/noirc_evaluator/src/ssa_refactor/acir_gen/acir_ir/acir_variable.rs

```diff
@@ -271,6 +271,48 @@ impl AcirContext {
             }
         }
     }
+    /// Returns an `AcirVar` that is constrained to be lhs << rhs.
+    ///
+    /// We convert left shifts to multiplications, so this is equivalent to
+    /// lhs * 2^rhs.
+    ///
+    /// We currently maintain that rhs needs to be a constant
+    /// however this can be extended, see #1478.
```

Comment:
```suggestion

    /// Returns an `AcirVar` that is constrained to be `lhs << rhs`.
    ///
    /// We convert left shifts to multiplications, so this is equivalent to
    /// `lhs * 2^rhs`.
    ///
    /// We currently require `rhs` to be a constant
    /// however this can be extended, see #1478.
```